src/Controller/MovieController.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\Routing\Annotation\Route;
+
+class MovieController extends AbstractController
+{
+    /**
+     * @Route("/movie", name="movie")
+     */
+    public function index()
+    {
+        $movies = $this->getDoctrine()->getRepository('App:Movie')->findAll();
+        //dump($movies);
```

Comment:
You should remove those `dump` code. Useful in development mode only

Reply:
Yes in the end it will be deleted